test/sql/issue_373.sql

```diff
@@ -0,0 +1,42 @@
+begin;
+
+    create table "Account"(
+      id serial primary key,
+      name text not null
+    );
+
+    create table "EmailAddress"(
+      id serial primary key,
+      "accountId" int not null, -- note: no foreign key
+      "isPrimary" bool not null,
+      address text not null
+    );
+
+    comment on table "EmailAddress" is e'
+        @graphql({
+            "foreign_keys": [
+              {
+                "local_name": "addresses",
+                "local_columns": ["accountId"],
+                "foreign_name": "account",
+                "foreign_schema": "public",
+                "foreign_table": "Account",
+                "foreign_columns": ["id"]
+              }
+            ]
```

Comment:
nit:
While the user in issue 373 is using a comment directive to define the foreign key, that is not material to the issue and could be confusing. Could you swap this out for a native key?